ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/feemarket/ZeroBaseFeeMarket.java

```diff
@@ -38,4 +38,9 @@ public Wei computeBaseFee(
   public ValidationMode baseFeeValidationMode(final long blockNumber) {
     return ValidationMode.NONE;
   }
+
+  @Override
+  public boolean implementsDataFee() {
+    return true;
```

Comment:
This was a bug introduced in https://github.com/hyperledger/besu/pull/4992 which was removed in https://github.com/hyperledger/besu/pull/5276 so it needs removing again I think.